src/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import { Button as ChakraButton, ButtonProps } from "@chakra-ui/react"
+
+import { scrollIntoView } from "../utils/scrollIntoView"
+
+export interface IProps extends ButtonProps {
+  toId?: string
+}
+
+const Button: React.FC<IProps> = ({ toId, children, ...props }) => {
```

Comment:
I'm calling this component `Button` bc it is the same Chakra button but with the scrolling feature. This way, you will just use this button component in the codebase.
```ts
import Button from 'components/Button'
```

On the other hand, we could break them out by calling this component `ButtonScroll` and using it only where we need to use the auto-scrolling.
```ts
import ButtonScroll from 'components/ButtonScroll'

// for the rest, we use the native chakra button
import Button from 'chakra/Button'
```


I just wanted to check if you have a preference.